app/src/test/java/org/simple/clinic/user/UserSessionTest.kt

```diff
@@ -219,6 +227,9 @@ class UserSessionTest {
       if (syncWillFail) Completable.error(RuntimeException()) else Completable.complete()
     }
     whenever(ongoingLoginEntryRepository.clearLoginEntry()).thenReturn(Completable.complete())
+    whenever(facilityRepository.associateUserWithFacilities(any(), any(), any())).thenReturn(Completable.complete())
```

Comment:
`any(), any(), any()` This is something we should get rid of eventually.

Reply:
Yes, I'll fix this when I move login and registration out into their own components as discussed offline.